code/registration.py

```diff
@@ -1440,28 +1548,41 @@ def singleplane_motion_correction(datainput: Path, output_dir: Path, debug: bool
     # Parse command-line arguments
     args = parser.parse_args()
     # General settings
-    datainput = Path(args.input_searchpath)
+    datainput = Path(args.input)
     output_dir = Path(args.output_dir)
     data_dir = Path("../data")
```

Comment:
Isn't `data_dir` the same as `datainput`?

Reply:
I thought I fixed that - must have been a different branch